news/2022-11-03-conda-survey1.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Tell Us How You Use Conda!"
+slug: "conda-survey1"
+tags: [conda]
+tease: "Conda users! We want you to help guide future directions for conda."
+external_url: "https://community.anaconda.cloud/t/tell-us-how-you-use-conda/44726/2"
```

Comment:
```suggestion
external_url: "https://community.anaconda.cloud/t/tell-us-how-you-use-conda/44726"
```

The way the URL was originally written didn't seem to take me to the correct page, but the above URL shows this which I think is what's supposed to show up:
![Screen Shot 2022-11-29 at 10 34 30 AM](https://user-images.githubusercontent.com/28930622/204572995-d2e2bb4f-9cc4-4610-ac13-f1c790803e63.png)

Reply:
When the `/2` is in the URL, this is where it leads me to:

![Screen Shot 2022-11-29 at 10 35 18 AM](https://user-images.githubusercontent.com/28930622/204573218-4bb07eb2-cd8f-4492-ab10-ad00ba073c01.png)